From f3f3f6f3697760408a84957a331836bc8cce5445 Mon Sep 17 00:00:00 2001
From: Hongxu Jia <hongxu.jia@windriver.com>
Date: Fri, 26 May 2017 14:10:54 +0800
Subject: [PATCH 17/65] invisible help button

WRLinux do not support help button, so we should not display it.

Do not display layout indicator neither.

Upstream-Status: Inappropriate [oe specific]

Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>

Disconnect F1 with event handler which triggers error since help button
has been removed.

Signed-off-by: Kai Kang <kai.kang@windriver.com>

---
 pyanaconda/ui/gui/__init__.py | 1 -
 widgets/src/BaseWindow.c      | 2 --
 2 files changed, 3 deletions(-)

diff --git a/pyanaconda/ui/gui/__init__.py b/pyanaconda/ui/gui/__init__.py
index 80dbb8eddd..c5ec94b258 100644
--- a/pyanaconda/ui/gui/__init__.py
+++ b/pyanaconda/ui/gui/__init__.py
@@ -450,7 +450,6 @@ class MainWindow(Gtk.Window):
         # Connect to mnemonics-visible to add the (F1) mnemonic to the button label
         if self._mnemonic_signal:
             self.disconnect(self._mnemonic_signal)
-        self._mnemonic_signal = self.connect("notify::mnemonics-visible", self._on_mnemonics_visible_changed, child)
 
         self._stack.set_visible_child(child.window)
 
diff --git a/widgets/src/BaseWindow.c b/widgets/src/BaseWindow.c
index 669c046530..c0822f37a2 100644
--- a/widgets/src/BaseWindow.c
+++ b/widgets/src/BaseWindow.c
@@ -415,8 +415,6 @@ G_GNUC_END_IGNORE_DEPRECATIONS
     gtk_grid_attach(GTK_GRID(win->priv->nav_area), win->priv->name_label, 0, 0, 1, 1);
     gtk_grid_attach(GTK_GRID(win->priv->nav_area), win->priv->distro_label, 1, 0, 2, 1);
     gtk_grid_attach(GTK_GRID(win->priv->nav_area), win->priv->beta_label, 1, 1, 1, 1);
-    gtk_grid_attach(GTK_GRID(win->priv->nav_area), win->priv->layout_indicator, 1, 2, 1, 1);
-    gtk_grid_attach(GTK_GRID(win->priv->nav_area), win->priv->help_button, 2, 1, 1, 2);
 
     /* Last thing for the main_box is a revealer for the info bar */
     win->priv->info_revealer = gtk_revealer_new();
-- 
2.7.4

